pop-api/integration-tests/src/local_fungibles.rs

```diff
@@ -367,6 +403,69 @@ fn transfer_works() {
 	});
 }
 
+#[test]
+fn transfer_from_works() {
```

Comment:
Could we add test situations for `AssetNotLive`. See other tests how to (freeze and destroy)